ui/types/@ember-data/store.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Store from 'ember-data/store';
```

Comment:
is this so we can reference `@ember-data/store` ?

Reply:
Yes it is. I'm not sure why eslint is not complaining about the import here since I enabled the rule again about importing from `ember-data` but in any case now we can import the type in the app like `import type Store from '@ember-data/store';`